src/essence/Basics/Layers_/Layers_.js

```diff
@@ -3791,6 +3799,107 @@ function parseConfig(configData, urlOnLayers) {
         //Otherwise return 0
         return 0
     }
+
+    // recurse through a STAC layer building sublayers
+    function getSTACLayers(d) {
+        let stac_data
+        const urlPieces = d.url.split(':')
```

Comment:
I had to change this line to 
```javascript
const urlPieces = d.url.split(/:(.*)/s)
```
to get it to work, otherwise it splits all the `:` and ends up only querying `https`

Reply:
good find, I'll fix that

Reply:
fixed in [bcf697b](https://github.com/NASA-AMMOS/MMGIS/pull/669/commits/bcf697b90dfe2dfc1cba862122f7bd1904350d83)